src/workflows/methods/SnapshotActionMenu.tsx

```diff
@@ -12,17 +12,17 @@ export interface SnapshotActionMenuProps {
   /**
    * Whether the user is an owner of the workflow snapshot the actions in the
    * menu are for. Controls whether the following actions are enabled: edit
-   * permissions, delete snapshot
+   * permissions, delete snapshot, a.k.a. version in the UI
```

Comment:
Should we rename this component?

Reply:
https://github.com/DataBiosphere/terra-ui/pull/5210